backend/app/crud/__init__.py

```
@@ -35,6 +36,7 @@
     set_creds_for_org,
     get_creds_by_org,
     get_key_by_org,
+    get_provider_credential,
     update_creds_for_org,
```

Ruff (0.12.2)

`.credentials.update_creds_for_org` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)